test/output/callbacks/callbacks_test.py

```diff
@@ -49,15 +50,18 @@ def _save_result(test_record):
   def test_json(self, user_mock):
     user_mock.prompt.return_value = 'SomeWidget'
     record = yield self._test
-    json_output = StringIO()
+    if sys.version_info[0] < 3:
```

Comment:
This is suspicious and makes me think I screwed something up when migrating the attachment strings. Maintainers, any insights / opinions?